Cargo.toml

```diff
@@ -24,7 +24,7 @@ quil-rs = { version = "0.22.0", git = "https://github.com/rigetti/quil-rs", tag
 # and need to be updated together.
 ndarray = { version = "0.15.6", features = ["serde"] }
 numpy = "0.19"
-pyo3 = "0.19.2"
+pyo3 = { version = "0.19.2", features = ["extension-module"] }
```

Comment:
Is the `extension-module` feature needed here? IIRC we omit it here and instead include it in `build.rs` because having the feature here causes problems when building the Python library as a Rust target.

Reply:
Hm. No, not needed. I was having trouble getting tests to run, and the pyo3 docs suggested this change.